TrafficSimulator/Assets/Cam/CameraManager.cs

```diff
@@ -41,9 +51,10 @@ private void OnDestroy()
             InputHandler.Dispose();
         }
         
-        public void ToggleThirdPersonCamera()
+        public void ToggleFocusCamera()
         {
             SwitchActiveCamera(1);
```

Comment:
Do these ints correspond to the different cameras? If so, it should probably be an enum instead so it will be clearer. If you still need the int value, you could just cast from the enum.

If you need custom int values for the different cameras then that can be setup in the enum, something like this:
```
enum CameraType
{
    Focus = 0,
    Sample = 1,
    FirstPersonDriver = 123,
    Default = 999
}
```

Reply:
Fixed!